site/content/docs/5.0/examples/headers/index.html

```diff
@@ -157,7 +157,7 @@ <h1 class="visually-hidden">Headers examples</h1>
   <header class="py-3 mb-3 border-bottom">
     <div class="container-fluid d-grid gap-3 align-items-center" style="grid-template-columns: 1fr 2fr;">
       <div class="dropdown">
-        <a href="#" class="d-flex align-items-center col-lg-4 mb-2 mb-lg-0 link-dark text-decoration-none dropdown-toggle" id="dropdownNavLink" data-bs-toggle="dropdown" aria-expanded="false">
+        <a href="#" class="d-flex align-items-center col-lg-4 mb-2 mb-lg-0 link-dark text-decoration-none dropdown-toggle" id="dropdownNavLink" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Get the dropdown menu containing navigation links">
```

Comment:
the best way to render an svg accessible : 

1. put a `<title>` inside the `svg` as the first child with the description inside with an id
2. add an aria-labelledby="theid" onto the svg to link it
example: 

```
<svg role="img" aria-labelledby="unique-svg-title-id"> <title id="unique-svg-title-id">the description here</title><....></svg>
```

And next propose a fix to bootstrap :-)